### PR TITLE
Fix readability of tutorials in dark mode

### PR DESCRIPTION
**Pull request by kushalbakshi**
This PR fixes the readability of the Jupyter Notebook outputs in the `Using the json type` tutorials. This resolves [issue 1078](https://github.com/datajoint/datajoint-python/issues/1078). 

Below are the screenshots from local testing after applying changes to the code as seen in this PR: 
Dark mode - the font in the tables previously appeared white, and was therefore, not readable. See linked issue above for images.

image

Light mode - there was no issue here previously and remains the same as before.

image

### CODE REVIEW

File: docs/src/.overrides/assets/stylesheets/extra.css [hunk 1]
```diff
@@ -91,3 +91,7 @@ html a[title="YouTube"].md-social__link svg {
     /* previous/next text */
     /* --md-footer-fg-color: var(--dj-white); */
 }
+
+[data-md-color-scheme="slate"] .jupyter-wrapper .Table td {
+    color: var(--dj-primary);
```
Comment (CBroz1) [inline docs/src/.overrides/assets/stylesheets/extra.css:96]: Do we perhaps want this to be dark background with light text?

Comment (kushalbakshi) [reply docs/src/.overrides/assets/stylesheets/extra.css:96]: Yes - we can certainly do that. Here's an example. We can go with any background color and text color combination now but I've picked `dj-black` and `dj-primary` in the example below. If that looks better, I can push a commit with the update. 

image

Comment (CBroz1) [reply docs/src/.overrides/assets/stylesheets/extra.css:96]: I worry that this is overuse of our blue. I think I'd flip what you have - Blue on black for the header (using blue or bold for primary keys), black on grey for the contents.

Comment (CBroz1) [reply docs/src/.overrides/assets/stylesheets/extra.css:96]: I'd suggest a dev branch on your fork if you want to trying things out as committed. Maybe you're doing this already, but, you'll need to rerun the notebook in question to refresh any info coming from `preview.py`. CSS does have some tools for prioritizing one rule over another

Lmk if you want to schedule a co-coding session to work on the table header

Comment (kushalbakshi) [reply docs/src/.overrides/assets/stylesheets/extra.css:96]: Ah, okay. I was not running the notebook again. One reservation of using `preview.py` for this is that I don't believe `preview.py` makes any distinction between light and dark mode at this point. It's probably not much more effort to include it but it adds time for implementing and testing. So before I dive into it, I want to ensure we'll be using this approach on the site for future tutorials as well? If so, I'm happy to proceed with developing it on a dev branch and potentially scheduling a co-coding session.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## Release notes
 
+### 0.14.1 -- Feb 21, 2023
+- Fixed - .ipynb output in tutorials is not visible in dark mode ([#1078](https://github.com/datajoint/datajoint-python/issues/1078) PR [#1080](https://github.com/datajoint/datajoint-python/pull/1080))
```
Comment (guzman-raphael) [inline CHANGELOG.md:4]:
```suggestion
- Fixed - `.ipynb` output in tutorials is not visible in dark mode ([#1078](https://github.com/datajoint/datajoint-python/issues/1078)) PR [#1080](https://github.com/datajoint/datajoint-python/pull/1080)
```

File: docs/src/.overrides/assets/stylesheets/extra.css (hunk 1 above)
Comment (guzman-raphael) [reply docs/src/.overrides/assets/stylesheets/extra.css:96]: How about we set the text color of the table to be `--dj-black`? Think that might be cleaner and more consistent.

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
@kushalbakshi Thanks for patching this! :muscle: 

Dark mode users (:raising_hand_man:) will certainly appreciate this improvement. Providing some small suggestions but otherwise agree with the approach here.

**Comment by kushalbakshi** [comment]
Hi @guzman-raphael, I'm finally getting the chance to get back on this one. 

Here is one example each of black background with white text, and black text with white background. Let me know what you prefer. I think the black background is cleaner in dark mode, but it's up to you. 

black background

white background

**Comment by guzman-raphael** [comment]
Thanks @kushalbakshi. That is an interesting option. :thinking: 

Just discussed this with @kabilar to get his opinion and he agreed with me that the white background one is cleaner. Let's proceed with that.

**Comment by kushalbakshi** [comment]
> Thanks @kushalbakshi. That is an interesting option. 🤔
> 
> Just discussed this with @kabilar to get his opinion and he agreed with me that the white background one is cleaner. Let's proceed with that.

Sounds good! The changes have been tested and committed here. 

Something to note - the docs container would not build without the `typing-extensions` package. There was a `ModuleNotFoundError`. I've added that to `docs/.docker/pip_requirements.txt`. Here's the error message I received prior to adding this requirement: 
```
docs-docs-1  |   File "/opt/conda/lib/python3.9/site-packages/mkdocstrings/plugin.py", line 41, in <module>
docs-docs-1  |     from typing_extensions import ParamSpec
docs-docs-1  | ModuleNotFoundError: No module named 'typing_extensions'
docs-docs-1 exited with code 1
```

**Review by guzman-raphael — APPROVED** [review]

**Pull request merged**